src/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -42,15 +42,35 @@ public int DoRun(string[] args)
                     RegisterForParentProcessExit(dotnetTestParams.ParentProcessId.Value);
                 }
 
-                var projectContexts = CreateProjectContexts(dotnetTestParams.ProjectPath);
-
-                var projectContext = projectContexts.First();
-
-                var testRunner = projectContext.ProjectFile.TestRunner;
+                var projectContexts = CreateProjectContexts(dotnetTestParams.ProjectPath, dotnetTestParams.Runtime);
+                var ranTests = false;
+                foreach (var projectContext in projectContexts)
+                {
+                    if (dotnetTestParams.Framework != null && dotnetTestParams.Framework != projectContext.TargetFramework)
+                    {
+                        continue;
+                    }
+
+                    ranTests = true;
+                    var testRunner = projectContext.ProjectFile.TestRunner;
+                    var dotnetTestRunner = _dotnetTestRunnerFactory.Create(dotnetTestParams.Port);
+                    var result = dotnetTestRunner.RunTests(projectContext, dotnetTestParams);
+                    if (result != 0)
+                    {
+                        return result;
```

Comment:
I think continue running tests for all tfms.  Just like Xunit runs all test methods/classes even if one fails.  As a dev, I like to know all the tests I need to fix.  I don't like having to fix a few, and then run again to find more tests failing